src/pip/_internal/_output

```diff
@@ -0,0 +1,1189 @@
+from __future__ import absolute_import
```

Comment:
Looks like an extra file made it into the set of changes. Can we remove this?

Reply:
Whops